package.json

```diff
@@ -3,10 +3,12 @@
   "version": "0.1.0",
   "description": "USFS NRM App",
   "devDependencies": {
+    "husky": "^7.0.1",
     "pa11y-ci": "^2.4.2"
   },
   "scripts": {
-    "test:a11y": "pa11y-ci --config a11y/scripts/main.json"
+    "test:a11y": "pa11y-ci --config a11y/scripts/main.json",
+    "prepare": "husky install"
```

Comment:
**suggestion:** Can we document somewhere in a README how a new developer could choose to use this?

Reply:
That would be helpful. We are not certain what the new developers have used so a small descriptions would probably be helpful.

Reply:
Added documentation and reworked dependencies to make this opt-in.